sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -413,7 +413,7 @@ public partial class CommunityGalleryImageData : Azure.ResourceManager.Compute.M
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Compute.Models.GalleryImageFeature> Features { get { throw null; } }
         public Azure.ResourceManager.Compute.Models.HyperVGeneration? HyperVGeneration { get { throw null; } }
         public Azure.Core.ResourceIdentifier Id { get { throw null; } }
-        public Azure.ResourceManager.Compute.Models.GalleryImageIdentifier Identifier { get { throw null; } }
+        public Azure.ResourceManager.Compute.Models.CommunityGalleryImageIdentifier Identifier { get { throw null; } }
```

Comment:
Well looks like this change is unnecessary for dotnet as well - this property does not have a setter, therefore the setters and the full parameter ctor on `GalleryImageIdentifier` are meaningless for this property. Therefore if we do not change this, it works as before perfectly.
Therefore this is definitely an unnecessary change for dotnet

Reply:
well after some thoughts I believe our previous might not work if the response returns null value for any of these properties - we might get NRE during the deserialization because the previous model only have one constructor and it is expecting all those values are not null.
But still I think this change in the swagger is an overkill, we do not need to introduce such a big change - could we remove the required flags on `GalleryImageIdentifier` model?

Reply:
please see my email on what I think creating this new model is necessary @ArcturusZhang